examples/06_products/05_get_product_media/query.graphql

```diff
@@ -29,11 +29,11 @@ fragment mediaFieldsByType on Media {
   ...on ExternalVideo {
     id
     host
-    embeddedUrl
+    originUrl
   }
```

Comment:
Maybe keep the embeddedUrl (new name: embedUrl). I tested with a youtube video as the externalVideo in my test store. The originUrl this query return is "https://youtu.be/yE0NGXZ_HBs" while the embedUrl is "https://www.youtube.com/embed/yE0NGXZ_HBs". This attribute is still very helpful to the dev

Reply:
Never mind, forget what I said. It's not necessary